index.js

```diff
@@ -89,7 +91,20 @@ class Queue extends EventEmitter {
       await wait(this.waitDelay);
     } else {
       this.emit('process', job.value);
-      await this.runJob.bind(this)(job.value);
+      const b = this.runJob.bind(this);
```

Comment:
seems like this code should be inside runJob since that is already catching for errors